frontend/src/plugins/impl/DataTablePlugin.tsx

```diff
@@ -250,13 +257,27 @@ export const LoadingDataTableComponent = memo(
     }, [sorting, search, filters, searchQuery, props.fieldTypes, props.data]);
 
     // Column summaries
-    const { data: columnSummaries, error: columnSummariesError } =
-      useAsyncData(() => {
+    const { data: columnSummariesData, error: columnSummariesError } =
+      useAsyncData<ColumnSummaries>(() => {
         if (props.totalRows === 0) {
           return Promise.resolve({ summaries: [] });
         }
+        // Clear the summary first to avoid inconsistency with table
+        setSummaries({ summaries: [] });
```

Comment:
i think it is fine to show the previous summaries while the new ones are loading, otherwise it might flicker a bunch

Reply:
Fixed. Thanks!